addons/mail/data/ir_cron_data.xml

```diff
@@ -75,5 +75,14 @@
             <field name="code">model._unmute()</field>
             <field name="state">code</field>
         </record>
+
+        <record id="ir_cron_discuss_users_settings_unmute" model="ir.cron">
```

Comment:
Why a new cron? This can be part of the autovacuum cron.

Reply:
it is not a daily autovacuum cron. it will be called when the user sets a duration for the mute and the cron will be called after the duration.

Reply:
Never mind. The cron is triggered as soon as possible explicitly.